.github/workflows/bootzooka-scala-ts-ci.yml

```diff
@@ -1,18 +1,21 @@
 name: Bootzooka CI
 
 on:
+  pull_request:
   push:
+    branches:
+      - master
     paths-ignore:
       - "helm/**"
-  pull_request:
+  release:
+    types:
+      - released
     paths-ignore:
       - "helm/**"
 
 jobs:
-  scala-ts-tests:
-    # run on external PRs, but not on internal PRs since those will be run by push to branch
-    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
this was quite useful as when we were doing PRs from branches in this repository (not from forks), we got double builds (one for the PR, one for the in-repository branch). Is this problem solved somehow else?

Reply:
Sure, but please note I also added explicite branch name in `on:`.
So now this workflow is triggered:
- on push to master branch
- on PR (create, update etc)
- on GH release.
So no double builds (unless unlikely event when someone creates PR with master as source branch)